### ISSUE TITLE
[Improvement]: Add Unload Menu To Chat CMD

### ISSUE THREAD

**Issue by ToXicVibezz** [Yimura/YimMenu#2580]
### Which feature

I think having the option to unload the menu from chat commands would be a nice feature (I use this quite often). 

### Reason

There is a FastQuit Option .. I think that adding this option would give some users a easier way to unload Yim.

### Additional context

``#include "backend/command.hpp"
#include "backend/looped_command.hpp"
#include "script.hpp"
#include "fiber_pool.hpp"

namespace big
{
	class unload_menu : command
	{
		using command::command;

		virtual void execute(const command_arguments&, const std::shared_ptr<command_context> ctx) override
		{
			g_fiber_pool->reset();
			g_fiber_pool->queue_job([] {
				for (auto& command : g_looped_commands)
					if (command->is_enabled())
						command->on_disable();

				g_running = false;
			});
		}
	};

	unload_menu g_unload_menu("unload", "Unload Menu", "Unload The Menu From GTA Without Game Destruption.", 0);
}``

**Comment by ToXicVibezz** [comment]
... i mean its all there just not in ` `

**Comment by ghost** [comment]
Not sure why you would want that as it could in theory allow anyone who has chat command access to run that command and unload YOUR menu (as well). Unless you add a check that only allows yourself then I fail to see a valid use case where you would want this.

**Comment by Yimura** [comment]
> Not sure why you would want that as it could in theory allow anyone who has chat command access to run that command and unload YOUR menu (as well). Unless you add a check that only allows yourself then I fail to see a valid use case where you would want this.

There’s per command permission system, you can limit the command appropriately. So only if you give a specific player the permission to unload then yeah he’ll be able to unload.

**Comment by ToXicVibezz** [comment]
> > Not sure why you would want that as it could in theory allow anyone who has chat command access to run that command and unload YOUR menu (as well). Unless you add a check that only allows yourself then I fail to see a valid use case where you would want this.  Also I have it as a hotkey . Could make a ImGui::OpenPopup , similiar to have gta_data cache works.

> 
> There’s per command permission system, you can limit the command appropriately. So only if you give a specific player the permission to unload then yeah he’ll be able to unload.

Yes.. What I had Actually meant was, I personally use this as a way i can easily type `unl` and it will unload the menu .. Also I use unstuck = give script host , interior tp/interior kick (played with adding suicide or new session). I will test to see if this can be abused. Ive use the cmd executor , ive actually never tried chat cmds with Yim.

**Comment by ToXicVibezz** [comment]
> Not sure why you would want that as it could in theory allow anyone who has chat command access to run that command and unload YOUR menu (as well). Unless you add a check that only allows yourself then I fail to see a valid use case where you would want this.

Yes i didnt word this right .. I wouldnt set it lower than of Admin level. .Which i didnt think would allow others to use it. I dont think it would be to hard to add a check. Can this be done with  ,, self::ped / the `is_player_our_boss` ?

**Comment by ToXicVibezz** [comment]
Im not sure if you can see what i also said (after my edit.. I cant) .. I said i have it as a hotkey as well and wonder if using something like Yim's first time cache ImGui::OpenPopup could be used to prompt the user if theyre sure they want to unload with .. tho ifeel like this might just be a feature that i find usefull more than others.

**Comment by maybegreat48** [comment]
Game chat doesn't work during loading screens, in the clouds, in singleplayer, in creator, etc. I'd just stick with the hotkey if you really have to unload quickly

**Comment by GreySideMM** [comment]
There is no unload hotkey or command. You probably mean "rage quit". I guess unload could be added but honestly idk why you would want to unload it that quickly?
